ParcelInvoiceMaker/ParcelInvoiceMaker/Error/ParcelInformationError.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ParcelInformationError.swift
+//  ParcelInvoiceMaker
+//
+//  Created by MIN SEONG KIM on 2024/01/25.
+//
+
+import Foundation
+
+// 택배정보 입력 에러
+enum ParcelInformationError: Int, Error {
+    case addressError = 0,
+         nameError,
+         mobileError,
+         deliveryCostError
+}
```

Comment:
이넘은 ,로 사용하기 보다는 아래처럼 정의를 많이 하는거 같아요
```suggestion
enum ParcelInformationError: Int, Error {
    case addressError = 0
    case nameError
    case mobileError
    case deliveryCostError
}
```